package.json

```diff
@@ -58,7 +58,7 @@
   "module": "out/replicache.mjs",
   "types": "out/replicache.d.ts",
   "bin": {
-    "get-license": "./out/get-license.cjs"
+    "get-replicache-license": "./out/get-license.cjs"
```

Comment:
I know this is just "bike shedding" but I would prefer if this was:

```
"bin": {
  "replicache": "./out/cli.js"
}
```

because then one can do:

```
npx replicache get-license
```

If the name of the bin is the same as the npm package you can run this from anywhere and npm will download it as needed.

For example you can do:

```
npx esbuild --help
```

or

```
npx playwright help install
```

from anywhere

Reply:
I'm not a big fan of repainting a shed that works just fine, esp since we haven't launched yet, but ok will make this change.